core/src/main/web/app/styles/modal.scss

```diff
@@ -8,6 +8,10 @@ html {
     border-bottom-width: 0;
   }
 
+  .modal-open.has-scroll {
+    margin-right: 17px;
+  }
+
```

Comment:
hi, we have different scrollbar width for ambiance theme. so this style should also be overridden